packages/patternfly-4/react-inline-edit-extension/src/components/InlineEdit/utils/formatters/inlineEditFormatterFactory.d.ts

```diff
@@ -23,8 +21,7 @@ export interface InlineEditFormatterFactoryProps {
   renderEdit?(value: ReactNode, additionalData: AdditionalData, computedData: ComputedData): ReactNode;
 }
 
-export interface InlineEditFormatterFactory {
-  (props: InlineEditFormatterFactoryProps): Formatter;
-}
+export type InlineEditFormatterFactory = (props: InlineEditFormatterFactoryProps) => Formatter;
```

Comment:
This is a breaking change. 'interface' to 'type'

Reply:
Reverted